lib/testcase/sparql/TestCaseQueryEvaluation.ts

```diff
@@ -263,6 +266,8 @@ export class TestCaseQueryEvaluation implements ITestCaseSparql {
       throw new Error(`Invalid query evaluation
   Query: ${this.queryString}
 
+  Data: ${this.dataUri}
```

Comment:
`this.dataUri` can be null, so I would suggest replacing this with something like `${this.dataUri || 'none'}`

Reply:
Additionally, I would append something like

    this.dataGraph ? `(named graph: ${this.dataGraph})` : ''

As this data might be encapsulated in a specific named graph.

Reply:
@rubensworks replaced the commit with this